src/Proxy/RequisitionStoredProcedures.cs

```diff
@@ -420,5 +420,52 @@ public async Task<ProcessResult> CreateRequisitionLines(int reqNumber, int? seri
 
             return new ProcessResult(successParameter.Value?.ToString() == "1", messageParameter.Value?.ToString());
         }
+
```

Comment:
again if this calls stores_oo.can_book_req I've already rewritten this logic in RequisitionHeader.CanBookReq c# when writing book req

Reply:
will change